sonar-project.properties

```diff
@@ -0,0 +1,4 @@
+# Configuration file for SonarCloud static code analysis
```

Comment:
Do you think you could give a quick explanation of what SonarCloud is somewhere - possibly in the readme or wiki. Feel free to disregard if you think this is too basic.

Reply:
I think I will leave it because this file states its for static code analysis and then the only time it's being used is on checks which provide context about what it does. Looking at the output that it leaves on PRs  its clear it looks for vulnerabilities and duplication and code smells etc. But that's my opinion after using it on and off for the last 3 years, what are your thoughts? @rosepearson